db/structs.go

```diff
@@ -226,6 +226,9 @@ type ConnectionCodes struct {
 	ConnectionString string     `json:"connection_string"`
 	IsUsed           bool       `json:"is_used"`
 	DateCreated      *time.Time `json:"date_created"`
+	Pubkey           string     `json:"pubkey"`
```

Comment:
Hi @aliraza556 we don't need `Pubkey`, `RouteHint` and `SatsAmount` in the ConnectionCodes struct